application/templates/dashboards/planned_pages.html

```diff
@@ -56,12 +56,12 @@
     <table id="measure-table" class="measure-table">
 
         <thead>
-        <tr>
-            <th aria-sort="none">Measure</th>
-            <th class="numeric" aria-sort="descending">Dept</th>
-            <th class="numeric" aria-sort="none">Stage</th>
-            <th class="numeric" aria-sort="none">Type</th>
-        </tr>
+            <tr>
+                <th class="column-width-40" aria-sort="none">Page</th>
```

Comment:
I'm not convinced we should set these as percentages, as that might not work on smaller screens. May be better to have fixed widths for the last three columns, and let the first one take up the rest? Although that might need adjusting for smaller screens too...

Reply:
It looks OK to me in browser as I squash it up, and in "device mode" for phone simulation.  The problem with fixed width is that they won't grow for wider screens where there is space for the extra stuff?

I don't know... I've had a bit of a play and fixed width doesn't really seem any better than this.